addons/payment/views/payment_provider_views.xml

```diff
@@ -59,24 +59,24 @@
                         </div>
                     </div>
                     <div id="provider_creation_warning" attrs="{'invisible': [('id', '!=', False)]}" class="alert alert-warning" role="alert">
-                        <strong>Warning</strong> Creating a payment acquirer from the <em>CREATE</em> button is not supported.
+                        <strong>Warning</strong> Creating a payment provider from the <em>CREATE</em> button is not supported.
                         Please use the <em>Duplicate</em> action instead.
                     </div>
                     <group>
                         <group name="payment_state">
-                            <field name="provider" groups="base.group_no_one" attrs="{'readonly': [('id', '!=', False)], 'invisible': [('module_id', '!=', False), ('module_state', '!=', 'installed')]}"/>
+                            <field name="code" groups="base.group_no_one" attrs="{'readonly': [('id', '!=', False)], 'invisible': [('module_id', '!=', False), ('module_state', '!=', 'installed')]}"/>
                             <field name="state" widget="radio" attrs="{'invisible': [('module_state', '=', 'uninstalled')]}"/>
                             <field name="company_id" groups="base.group_multi_company" options='{"no_open":True}'/>
                         </group>
                     </group>
                     <notebook attrs="{'invisible': ['&amp;', ('module_id', '!=', False), ('module_state', '!=', 'installed')]}">
-                        <page string="Credentials" name="acquirer_credentials" attrs="{'invisible': ['|', ('provider', '=', 'none'), ('show_credentials_page', '=', False)]}">
-                            <group name="acquirer"/>
+                        <page string="Credentials" name="provider_credentials" attrs="{'invisible': ['|', ('code', '=', 'none'), ('show_credentials_page', '=', False)]}">
+                            <group name="code"/>
```

Comment:
why `code` as name? :thinking: 
maybe something like `provider_keys` would be more clear :thinking:
(for finetuning pr ofc)